app/pages/adduser/contextCreator.js

```diff
@@ -2,7 +2,7 @@ import manifest from './manifest.json';
 
 export const getContext = data => ({
   ...manifest,
-  orgId: data.organisationId,
+  organisationId: data.organisationId,
```

Comment:
😨 if we are going to change this. Can we change `orgName` too please

Reply:
sure